src/AzureStack/Admin/BackupAdmin/Backup.Admin/Properties/AssemblyInfo.cs

```diff
@@ -14,5 +14,5 @@
 [assembly: AssemblyCulture("")]
 [assembly: NeutralResourcesLanguage("en")]
 
-[assembly: AssemblyVersion("0.1.0.0")]
-[assembly: AssemblyFileVersion("0.1.0.0")]
+[assembly: AssemblyVersion("0.1.1.0")]
```

Comment:
Please do not change the `AssemblyVersion` here, only `AssemblyFileVersion` needs to be updated